src/index.js

```diff
@@ -6,13 +6,17 @@ import './css/base.scss';
 import fetcher from './fetch.js';
 import User from './User.js';
 import domUpdates from './DOM-updates.js';
+const moment = require('moment');
+
 
```

Comment:
I look around, and all I see, is empty lines.

Reply:
Empty lines are just lines that I haven't filled yet